### DISCUSSION TITLE
BottonNavigationView

### DISCUSSION THREAD

**Discussion by JOCKER805** [Sketchware-Pro/Sketchware-Pro#738] [Q&A] [answered]
----------
1. ERROR in /storage/emulated/0/.sketchware/mysc/602/app/src/main/java/com/my/newproject/MainActivity.java (at line 24)
	import com.google.android.material.bottomnavigation.BottomNavigationView;
	       ^^^^^^^^^^
The import com.google cannot be resolved
----------
2. ERROR in /storage/emulated/0/.sketchware/mysc/602/app/src/main/java/com/my/newproject/MainActivity.java (at line 25)
	import com.google.android.material.bottomnavigation.BottomNavigationView.OnNavigationItemSelectedListener;
	       ^^^^^^^^^^
The import com.google cannot be resolved
----------
3. ERROR in /storage/emulated/0/.sketchware/mysc/602/app/src/main/java/com/my/newproject/MainActivity.java (at line 34)
	private BottomNavigationView bottomnavigation1;
	        ^^^^^^^^^^^^^^^^^^^^
BottomNavigationView cannot be resolved to a type
----------
4. ERROR in /storage/emulated/0/.sketchware/mysc/602/app/src/main/java/com/my/newproject/MainActivity.java (at line 45)
	bottomnavigation1 = findViewById(R.id.bottomnavigation1);
	^^^^^^^^^^^^^^^^^
BottomNavigationView cannot be resolved to a type
----------
5. ERROR in /storage/emulated/0/.sketchware/mysc/602/app/src/main/java/com/my/newproject/MainActivity.java (at line 47)
	bottomnavigation1.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
	^^^^^^^^^^^^^^^^^
BottomNavigationView cannot be resolved to a type
----------
6. ERROR in /storage/emulated/0/.sketchware/mysc/602/app/src/main/java/com/my/newproject/MainActivity.java (at line 47)
	bottomnavigation1.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
	                                                          ^^^^^^^^^^^^^^^^^^^^
BottomNavigationView cannot be resolved to a type
----------
7. ERROR in /storage/emulated/0/.sketchware/mysc/602/app/src/main/java/com/my/newproject/MainActivity.java (at line 49)
	public boolean onNavigationItemSelected(MenuItem item) {
	               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
The method onNavigationItemSelected(MenuItem) of type new OnNavigationItemSelectedListener(){} must override or implement a supertype method
----------
7 problems (7 errors)

**Comment by j-px** [comment, marked as answer]
You need to enable AppCompat in Library Manager to be able to use BottomNavigationView.
